tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -548,10 +548,6 @@ def generate_files(line_list, args):
             files_list.append(
                 "<file src=" + '"' + os.path.join(args.native_build_path, "onnxruntime.dll") + runtimes + " />"
             )
-            if include_pdbs and os.path.exists(os.path.join(args.native_build_path, "onnxruntime.pdb")):
```

Comment:
I think it was removed because we could not build the nuget for ovep with windows debug. But I see here that there is no check for openvino.  so this is a mistake. Will get it resolved .